client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,53 @@
+.scoreboard {
+  box-sizing: border-box;
+  background: #ffffff;
+  color: #ffffff;
+  border: 2px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.scoreboard table {
+  border:  none;
+  border-collapse: separate;
+  border-spacing: 0;
+  display: block;
+  overflow-x: auto;
+  overflow-y: auto;
+}
+
+.scoreboard .firstRow.firstCol {
+  z-index: 2;
+}
+
+.scoreboard .firstRow {
+  position: sticky;
+  z-index: 1;
+  top: 0;
+  background-color: #ddd;
+}
+
+.scoreboard .firstCol {
+  position: sticky;
+  left: 0;
+  background-color: #ddd;
+}
+
+.scoreboard tbody td {
+  border-bottom: 1px solid #ddd;
+  border-right: 1px solid #ddd;
+  color: #333;
+  padding: 5px;
+  text-shadow: 1px 1px 1px #fff;
+  text-align: right;
+  white-space: nowrap;
+  background-color: #f8f8f8;
+}
+
+.scoreboard .totalsLine {
```

Comment:
The first column of the totalsLine needs a higher z-index so it doesn't slide behind the other totals as you scroll right, e.g.
```css
.scoreboard .totalsLine.firstCol {
  z-index: 1;
}
```
Setting a z-index: 1 seems appropriate as it shouldn't go above the top row first column title if the scoreboard gets really tiny.

Reply:
Thanks for catching this.